site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -348,6 +348,37 @@ Workloads as possible, preferring Workloads with these characteristics:
 - Workloads with the lowest priority.
 - Workloads that have been admitted more recently.
 
+## FlavorFungibility
+
+When there is not enough nominal quota of resources in a ResourceFlavor, the incoming workload can borrow or preempt in ClusterQueue or Cohort. 
+
+By default the incoming workload will stop trying the next flavor if it can get enough resource by borrowing. 
```

Comment:
```suggestion
By default, the incoming workload  stops trying the next flavor if the workload can get enough borrowed resources. 
```

I'm making a lot of tech assumptions. Feel free to disregard if the meaning was affected